src/_data/languages_base.json

```
@@ -71,6 +73,7 @@
     "rssFeedHeading": "RSSフィード",
     "rssFeedParagraph": "OWAの最新ニュースを購読する。",
     "siteFooterPoweredBy": "このサイトはNetlifyを利用しています",
-    "skipLinkText": "コンテンツにスキップする"
+    "skipLinkText": "コンテンツにスキップする",
+    "translationNotAvailable": "残念ながら、このページはまだご希望の言語に翻訳されていません。翻訳にご協力いただける場合は、OWA discord の #translation にご参加ください。"
```

@shuuji3 @AXT-AyaKoto could you please confirm if this translation to Japanese needs any changes?

---

@jessbudd Could you update to the following text?

```suggestion
    "translationNotAvailable": "残念ながら、このページはまだ日本語に翻訳されていません。翻訳にご協力いただける場合は、OWA Discord の #translation にご参加ください。"
```

To make it a more natural translation, I replaced "preferred language" with "Japanese" instead of a literal translation. (Also, uppercased "d" in "discord")

---

Thanks so much!